ext.json

```diff
@@ -0,0 +1,24 @@
+---
+metatype: json
+---
+
+{
+  "identifier": "io.github.xthursdayx.sn-material-theme",
+  "name": "Material Theme",
+  "content_type": "SN|Theme",
+  "area": "themes",
+  "version": "1.0.6",
+  "description": "Material Theme for Standard Notes",
+  "url": "https://github.com/xthursdayx/sn-theme-material/dist/main.css",
+  "download_url": "https://github.com/xthursdayx/sn-theme-material/archive/1.0.6.zip",
```

Comment:
What is the reason behind using a tagged archive instead of using the current master.zip?

Reply:
I agree with you - it does just make more work to for the maintainer. I was just following the format that is being used by other themes in the self-hosted SN extension repo. However, from what I remember the build script doesn't actually use the download link anyway, just looks for and downloads the latest tagged release, which is why themes and extensions without tagged releases don't work with the repository. That is also the reason that I've gone through and updated a bunch of themes and created releases; so that they can be included in the self-hosted repo by default, rather than having to be added by the user later. So, that's a long way of saying that you're probably right - I don't think it matters whether it is a tagged archive or master.zip, so feel free to switch it back to master.zip if you'd like. I think that should still work. I'll also give it a try on my end later today or tomorrow to make sure that the self-hosted SN Extensions repo will still build correctly using master.zip rather than the tagged archive url.